superset/utils/core.py

```diff
@@ -1231,7 +1246,205 @@ def ensure_path_exists(path: str) -> None:
             raise
 
 
-def get_since_until(  # pylint: disable=too-many-arguments
+class EvalText:  # pylint: disable=too-few-public-methods
```

Comment:
Can we move these classes/functions to a new file? E.g. `superset/utils/time_range.py`

The `core.py` file already quite large and we start breaking it up instead of adding new things to it.

Reply:
I'm having some trouble with circular imports. :( `utils.core` does need to be broken down, I can help with some of the this work.

Reply:
OK. Let's add that to TODO then. Thanks for tackling this!